tests/containers/podman_netavark.pm

```diff
@@ -46,9 +45,7 @@ sub is_cni_in_tw {
 # but images build with older pre-installed podman come with cni
 # fresh install of sle-micro comes with netavark
 sub is_cni_default {
-    my $podman_version = get_podman_version();
-    return package_version_cmp($podman_version, '4.8.0') < 0 ||
-      (is_sle_micro('<6.0') && !check_var('FLAVOR', 'DVD-Updates'));
+    return (is_sle_micro('<6.0') && !check_var('FLAVOR', 'DVD-Updates'));
```

Comment:
```suggestion
    return (is_sle_micro('<6.0'));
```
I think the Flavor check can be omitted as well. On SLEM<6 CNI remains the default AFAICS.

Reply:
@mloviska You added this condition. Do you agree?